tgui/packages/tgui/interfaces/CassetteManager.tsx

```diff
@@ -62,10 +64,17 @@ const CassetteTable = ({
           <Table.Cell>{cassette.tape_name}</Table.Cell>
           <Table.Cell>{cassette.reviewed ? 'Yes' : 'No'}</Table.Cell>
           <Table.Cell>
+            <Button
+              color="bad"
+              onClick={() => onReview('delete_cassette', id)}
+              style={{ marginLeft: '0.6em' }}
+            >
+              <i className="fa fa-trash" />
```

Comment:
can't this just use `<Icon>`

Reply:
Does this work better? It all relatively looks the same. Icon alone doesnt seem great.

```suggestion
            <Button
              color="bad"
              icon="trash"
              onClick={() => onReview('delete_cassette', id)}
            />
```

Reply:
ah yeah i forgot about that lol, yah it prolly works

Reply:
It looks decent to me.